frontend/src/components/buttons/Button.tsx

```diff
@@ -5,13 +5,24 @@ import styled from "styled-components";
 import { Colors } from "../../styles";
 
 const Button = styled(AButton)`
-  /* TODO: Change colors */
   background-color: ${Colors.primary};
   color: ${Colors.onPrimary};
+  border-radius: 8px;
+  border: none;
+
+  &:focus{
```

Comment:
focus, hover, active i visited można wrzucić jako jeden styl. W sensie, żeby nie powielać tego samego stylowania kilka razy, to możesz zapisać to tak:

`
&:focus, &:hover, &: active {
     background-color: ${darken(0.2, Colors.primary)};
     color: ${Colors.onPrimary};
}
`

Nie ma potrzeby powielać kodu, który robi dokładnie to samo.